tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "include": ["src", "types"],
+  "exclude": ["node_modules", "**/*.spec.ts"],
```

Comment:
It's actually not that suitable to exclude `"**/*.spec.ts"`. It causes conflicts when using inherited types and such.

We're ignoring the `*.spec.[jt]s` files in the publishing stage anyway.